dask/dataframe/io/parquet.py

```diff
@@ -84,33 +85,51 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None):
     else:
         index_col = None
 
+    all_columns = columns or (pf.columns + list(pf.cats))
+    if isinstance(all_columns, list):
+        all_columns = tuple(all_columns)
+    if not isinstance(all_columns, tuple):
+        all_columns = (all_columns,)
+    if index_col and index_col not in all_columns:
+        all_columns = all_columns + (index_col,)
+
     # TODO: if categories vary from one rg to next, need to cope
     dtypes = {k: ('category' if k in (categories or []) else v) for k, v in
-              pf.dtypes.items() if k in columns}
+              pf.dtypes.items() if k in all_columns}
 
     meta = make_meta(dtypes)
     if index_col:
         meta = meta.set_index(index_col)
 
-    name = 'read-parquet-' + tokenize(pf, columns, categories)
-    dsk = {(name, i): (read_parquet_row_group, index_col, open, pf.row_group_filename(rg),
-                        rg, columns, categories, pf.helper, pf.cats)
+    if columns is not None and not isinstance(columns, (tuple, list)):
+        assert len(meta.columns) == 1
+        out_type = Series
```

Comment:
Is there also a test case where the one column chosen happens to be the index column? In that case, I suppose the output should also be a Series, but without any particular index.